libraries/AP_Rally/AP_Rally.h

```diff
@@ -52,6 +52,8 @@ class AP_Rally {
         }
         return (uint8_t)ret;
     }
+    void truncate(uint8_t num);
+    bool append(const RallyLocation &loc) WARN_IF_UNUSED;
```

Comment:
The return value could use a description, presumably returns true if we manage to write the rally point out?